server/data/collections/metadata/fso_income_synthetic_metadata.yaml

```diff
@@ -2,17 +2,29 @@ max_ids: 1
 columns:
```

Comment:
I think we should add a documentation page about our metadata format

Reply:
True. I can create an issue for it.